index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+  <head>
+<meta charset="UTF-8">
+    <meta name="viewport"content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <main id="main">
+    <h1 id="title">Ahmad Shah Baba</h1>
+   <p>
+ <div id="img-div">
+    <img 
+    id="image" 
+    src="C:\Users\Power link\Downloads\images.jpg" 
+    alt="Ahmad Shah Baba" 
+    class="centered-image"align="center";>
```

Comment:
- Your image is not showing in the browser and the problem is that you linked the image from your local machine which is wrong:
  - You have to add an external (online) link for your image.
  - Or if you want to add an image from your local machine then you have to add the image to your project folder and then add the link of the image folder in your img tag.